pytexit/test/test_functions.py

```diff
@@ -7,9 +7,20 @@
 
 import sys
 
-from pytexit import py2tex, simplify, uprint
+
+# adding the trunk below will allow this script to import the pytexit that is in the parent folder
+# instead of the one installed in your system's python site-packages folder
```

Comment:
I really do not understand why you would need this ? 

If you using the Pytexit developer version (from GitHub), you should be installing it with 
```pip install -e .```

This removes and replace the Python's site-package folder one. 

Isn't this enough ?

Reply:
Because I am working on a forked project, so pip install should not work and is out of my control. Plus I am relying on pip installed pytexit for another project.

what I did is simply clone my forked version on my desktop. So I can refer to my modified version for a test by simply running the test_functions.py in my own folder.

Reply:
I do not think this is the proper way to do it, and these modifications shouldn't be added in the source code anyway.

You should have only one repository, with many Git branches. Some branches will track your Fork `origin` (turner-eng/pytexit), and some other branches will track the `upstream` (erwanp/pytexit), for instance the latest Master version if needed (`git add remote` if needed)

When working on pytexit just checkout the corresponding branch. When working on your other projects just checkout the master branch again.